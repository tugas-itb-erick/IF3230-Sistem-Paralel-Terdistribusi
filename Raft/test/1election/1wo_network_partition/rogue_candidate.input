5
set_timeout 1 0
timestep
set_timeout 1 0
timestep
set_timeout 1 0
timestep
set_timeout 1 0
timestep
set_timeout 1 0
timestep
set_timeout 1 0
timestep
flush_outbox
flush_outbox
get_state 1
end_simulation