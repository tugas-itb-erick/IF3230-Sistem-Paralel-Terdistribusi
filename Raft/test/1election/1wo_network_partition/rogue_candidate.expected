S1 leader term:7 voted_for:1 commit_index:0 data:0 logs:T0[0]
